index.js

```diff
@@ -16,15 +17,13 @@ async function downloadGrype(version) {
   // TODO: when grype starts supporting unreleased versions, support it here
   // Download the installer, and run
   const installPath = await cache.downloadTool(url);
-  // Make sure the tool's executable bit is set
-  await exec.exec(`chmod +x ${installPath}`);
 
-  let cmd = `${installPath} -b ${installPath}_grype ${version}`;
+  let cmd = `sh ${installPath} -d -b ${installPath}_grype ${version}`;
```

Comment:
Can you tell me a bit about what this does and why it's necessary? I'm sure it has something to do with making the install script run on Windows ...

Reply:
This eliminates the need to `chmod +x` the file, since it's just running `sh` directly. Note the `exec.exec(`chmod +x ${installPath}`);` was removed. The `-d` flag is for the [install.sh debugging](https://github.com/anchore/grype/blob/main/install.sh#L25)